### PR TITLE
@eessex => [Publishing] Responsive Article Layout components

### PR DESCRIPTION
**Pull request by kanaabe**
- Light refactor of story organization
- Crude text component (just as a placeholder for your upcoming work)
- `Article` component - pass an article and render its layout
- Responsive `StandardLayout` and `FeatureLayout`
- `SectionContainer` wraps every section and is responsible for setting widths
- Bug with artwork caption keys
- Handle missing profile image more gracefully
- New `sections` folder now contains all our section components 

feature_article_reaction

standard_article_reaction

### CODE REVIEW

File: src/components/publishing/sections/sections.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+const Div: StyledFunction<StyledSectionsProps> = styled.div
+
+const StyledSections = Div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin: ${props => chooseMargin(props.layout)}
+  max-width: ${props => (props.layout === "standard" ? "780px" : "auto")};
```
Comment (eessex) [inline src/components/publishing/sections/sections.tsx]: Nice handling of the layouts here!

File: src/components/publishing/sections/sections.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    switch (section.type) {
+      case "image_collection":
+        return (
+          <SectionContainer key={i} layout={section.layout}>
+            <ImageCollection images={section.images} targetHeight={500} gutter={10} />
+          </SectionContainer>
+        )
+      case "image_set":
+        return (
+          <SectionContainer key={i}>
```
Comment (eessex) [inline src/components/publishing/sections/sections.tsx]: lots of repetition of the SectionContainer here, I wonder if the case statement could move inside the container?

**Comment by eessex** [comment]
Looks great, looking forward to see the newly organized stories.  Other than my comment about the SectionContainer this looks good to go.

File: src/components/publishing/sections/sections.tsx (hunk 2 above)
Comment (kanaabe) [inline src/components/publishing/sections/sections.tsx]: Good call! Also removed the switch statement in favor of object literals.

**Comment by kanaabe** [comment]
Alrighty, going to merge this then!

**Pull request merged**